go/fixchain/fix_and_log.go

```diff
@@ -89,7 +95,7 @@ func NewFixAndLog(fixerWorkerCount int, loggerWorkerCount int, errors chan<- *Fi
 		t := time.NewTicker(time.Second)
 		go func() {
 			for _ = range t.C {
-				log.Printf("fix-then-log whole chains: %d already done", fl.alreadyDone)
+				log.Printf("fix-then-log: %d whole chains queued, %d whole chains already done, %d chains sent", fl.queued, fl.alreadyDone, fl.chainsSent)
```

Comment:
random use of commas.

Also, must be some way to avoid saying "chains" and "whole chains" every time.